src/pybind/mgr/cephadm/module.py

```diff
@@ -97,31 +98,6 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.cleanup()
 
 
-def name_to_config_section(name):
-    """
-    Map from daemon names to ceph entity names (as seen in config)
-    """
-    daemon_type = name.split('.', 1)[0]
-    if daemon_type in ['rgw', 'rbd-mirror', 'nfs', 'crash']:
-        return 'client.' + name
-    elif daemon_type in ['mon', 'osd', 'mds', 'mgr', 'client']:
-        return name
-    else:
-        return 'mon'
-
-def assert_valid_host(name):
-    p = re.compile('^[a-zA-Z0-9-]+$')
-    try:
-        assert len(name) <= 250, 'name is too long (max 250 chars)'
-        parts = name.split('.')
-        for part in name.split('.'):
-            assert len(part) > 0, '.-delimited name component must not be empty'
-            assert len(part) <= 63, '.-delimited name component must not be more than 63 chars'
-            assert p.match(part), 'name component must include only a-z, 0-9, and -'
-    except AssertionError as e:
-        raise OrchestratorError(e)
```

Comment:
conflicts with #34005

Reply:
dropped in favor of moving this func into python-common